src/meshes/t8code_mesh.jl

```diff
@@ -446,35 +428,191 @@ function T8codeMesh(conn::Ptr{p8est_connectivity}; kwargs...)
 end
 
 """
-    T8codeMesh(meshfile::String, ndims; kwargs...)
+    T8codeMesh(meshfile::String, NDIMS; kwargs...)
```

Comment:
We typically use `ndims` as a standard variable name and `NDIMS` for type parameters.

Reply:
Fixed.